util/raclgen/lib.py

```diff
@@ -0,0 +1,51 @@
+# Copyright lowRISC contributors (OpenTitan project).
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+import hjson
+import sys
+from reggen.validate import check_keys
+
+# Required fields for the RACL hjson
+racl_required = {
+    'roles': ['l', 'List, specifying all RACL roles'],
+    'policies': ['g', 'Dict, specifying the policies of all RACL groups']
+}
+
+
+# Default configuration to render the RACL package for systems that don't use RACL but need the
+# type definitions
+DEFAULT_RACL_CONFIG = {
+    'nr_policies': 1,
+    'policies': {},
+}
+
+
+def parse_racl_config(config_path: str):
+    try:
+        with open(config_path, 'r') as f_racl_config:
+            racl_config = hjson.load(f_racl_config)
+    except OSError:
+        raise SystemExit(sys.exc_info()[1])
+
+    # TODO further error handling
```

Comment:
Some hint about the missing checks?

Reply:
I changed the comment a little bit and linked the issue for that. https://github.com/lowRISC/opentitan/issues/25690